src/main/java/com/github/webdriverextensions/newversion/FileExtractorImpl.java

```diff
@@ -176,6 +177,8 @@ private void extractZipFile(Path file, Path toDirectory) throws IOException {
                                 Files.copy(zipArchive, fileToExtract);
                             } else {
                                 Path fileToExtract = toDirectory.resolve(zipEntry.getName());
+                                if(!Files.exists(fileToExtract.getParent()))
```

Comment:
Please add curly braces to the if statement and a space between "if" and the opening parenthesis to comply with the current formatting.